### DISCUSSION TITLE
Event RTSP feed not responding

### DISCUSSION THREAD

**Discussion by vgibara** [tsightler/ring-mqtt#880] [General Discussion]
I have a ring doorbell.

So I red almost (!) everything in the documentation. I understand that an event feed is available. Replace _live with _event at the end of the feed url.

Well, I can't make it work. Server not responding. However, my live feed is working.

I'd like to have a video when an event occurs (motion or ding) only.

Thanks for your help !

**Comment by tsightler** [comment]
Event stream plays back previously recorded events stored in the Ring cloud and thus requires a subscription.  If you are wanting a stream from the active event, that would need to use the live camera.

**Reply by vgibara** [reply]
I do have a subscription, but can't find the event stream url... Or just replacing _live with _event doesn't work.

**Reply by tsightler** [reply]
What do the logs show?

**Reply by vgibara** [reply]
It's mostly this:

2024-07-30T22:12:42.097Z ring-mqtt [Porte] TypeError: Cannot read properties of undefined (reading 'items')
2024-07-30T22:12:42.097Z ring-mqtt [Porte] No event recording corresponding to Motion 1 was found in device event history
2024-07-30T22:13:47.522Z ring-mqtt [Porte] HTTPError: Response code 503 (Service Unavailable)
2024-07-30T22:13:47.522Z ring-mqtt [Porte] Failed to retrieve device event history from Ring API
2024-07-30T22:13:47.522Z ring-mqtt [Porte] TypeError: Cannot read properties of undefined (reading 'items')
2024-07-30T22:13:47.522Z ring-mqtt [Porte] No event recording corresponding to Motion 1 was found in device event history

**Reply by tsightler** [reply]
Hmm...OK, that's interesting.  I believe this may be an unacknowledged outage (Ring has a lot of those) as this indicates that ring-mqtt could not retrieve event history due to the event history API service being unavailable thus there's no way to play back video from those events.

I was able to confirm the same error on my side, but, even more important, it is reproducible via the official Ring web dashboard page as well, the browser fails to return a list of events there with the same 503 error and sometimes the dashboard page itself fails to load.  This feature is dependent on the same method the dashboard uses to play back video events (it's nearly a 1:1 copy of that functionality) so hopefully they will fix it soon.

**Reply by tsightler** [reply]
Actually, Ring has just acknowledged a major outage as of 10 minutes ago on status.ring.com.

![image](https://github.com/user-attachments/assets/f1e4e44a-d69c-4bc3-a07f-94bdedb563a4)

**Reply by vgibara** [reply]
Haha that's me. I try a new thing, and there's an outage... I'm cursed ! Lol

Thanks, I'll try in a few days.

**Reply by vgibara** [reply]
I confirm my problem was related to the outage. Everything is working now.

Thanks !
